.github/container/build-jax.sh

```diff
@@ -275,14 +266,22 @@ time python build/build.py \
     --enable_nccl=true \
     --bazel_options=--linkopt=-fuse-ld=lld \
     --bazel_options=--override_repository=xla=$SRC_PATH_XLA \
+    --output_path=${BUILD_PATH_JAXLIB} \
     $BUILD_PARAM
 
-popd
+# Make sure that JAX depends on the local jaxlib installation
+# https://jax.readthedocs.io/en/latest/developer.html#specifying-dependencies-on-local-wheels
+line="jaxlib @ file://${BUILD_PATH_JAXLIB}"
+if ! grep -xF "${line}" "${SRC_PATH_JAX}/build/requirements.in"; then
+    pushd "${SRC_PATH_JAX}"
+    echo "${line}" >> build/requirements.in
+    PYTHON_VERSION=$(python -c 'import sys; print("{}.{}".format(*sys.version_info[:2]))')
+    bazel run //build:requirements_dev.update --repo_env=HERMETIC_PYTHON_VERSION="${PYTHON_VERSION}"
```

Comment:
One thing I wasn't so sure is that, is this setting `HERMETIC_PYTHON_VERSION` really necessary? I thought by default it's gonna use the system's python version. Could you try that? Other than that, everything else looks good!

Reply:
https://jax.readthedocs.io/en/latest/developer.html#specifying-python-version says it's only set automatically via `build.py`, which doesn't have a way of triggering ` requirements_dev.update`. The `_dev` version is needed to avoid putting hashes in the requirements file, which blocks using an editable installation.